src/VisualStudio.Roslyn.SDK/Roslyn.SDK/Properties/launchSettings.json

```diff
@@ -1,9 +1,9 @@
 {
   "profiles": {
-    "Templates.VisualStudio.2017": {
+    "Templates.VisualStudio.2019": {
       "commandName": "Executable",
       "executablePath": "$(DevEnvDir)devenv.exe",
-      "commandLineArgs": "/rootsuffix RoslynDev /log"
+      "commandLineArgs": "/rootsuffix Roslyn /log"
```

Comment:
```suggestion
      "commandLineArgs": "/rootsuffix $(VSSDKTargetPlatformRegRootSuffix) /log"
```

Reply:
does launch settings inherit this variable?

Reply:
It does. It's super handy.